common/cuda_hip/factorization/cholesky_kernels.hpp.inc

```diff
@@ -110,25 +113,28 @@ template <int subwarp_size, typename IndexType>
 __global__
     __launch_bounds__(default_block_size) void cholesky_symbolic_factorize_kernel(
         IndexType num_rows, const IndexType* row_ptrs,
-        const IndexType* lower_ends, const IndexType* postorder_cols,
-        const IndexType* postorder, const IndexType* postorder_parent,
-        const IndexType* out_row_ptrs, IndexType* out_cols)
+        const IndexType* lower_ends, const IndexType* inv_postorder,
+        const IndexType* postorder_cols, const IndexType* postorder,
+        const IndexType* postorder_parent, const IndexType* out_row_ptrs,
+        IndexType* out_cols)
 {
     const auto row = thread::get_subwarp_id_flat<subwarp_size, IndexType>();
     if (row >= num_rows) {
         return;
     }
     const auto row_begin = row_ptrs[row];
+    const auto diag_postorder = inv_postorder[row];
```

Comment:
Could you remind me what's inv_postoder and postorder?

Reply:
postorder is the sequence of row numbers such that each child comes before its parents (and thus a permutation), inv_postorder is the inverse permutation

Reply:
postorder[i]: i-th node is postorder[i]-th in original graph
row will be considered as inv_postorder[row] in new graph
is postorder_cols like inv_postorder[]?
col[i] will be considered as postorder_col[i] in the new graph

Reply:
from build_postorder_cols, it seems to be.